hazelcast/src/main/java/com/hazelcast/cache/impl/AbstractCacheService.java

```diff
@@ -835,4 +866,17 @@ public void onClusterStateChange(ClusterState newState) {
             expManager.onClusterStateChange(newState);
         }
     }
+
+    private static class CacheConfigFuture extends SimpleCompletableFuture<CacheConfig> {
```

Comment:
maybe a short javaDoc to explain purpose of this future.

Reply:
Added javadoc